code/game/jobs/job/command/auxiliary/intel.dm

```diff
@@ -22,6 +22,11 @@
 			total_positions_so_far = positions
 	return positions
 
+/datum/job/command/intel/generate_entry_conditions(mob/living/carbon/human/H)
+	..()
+	if(!Check_WO())
+		H.nutrition = rand(NUTRITION_VERYLOW, NUTRITION_LOW)
```

Comment:
Hmm, since IOs would be hungry on the level of cryo marines, where would you expect them to get food? As I understand MREs are meant to be used when you deploy and get hungry, not as your starting source of food. And their closest source of food is what, lower medbay or squad preps? That's a long way to hobble. Why do you think this will improve the IO gameplay?

Also, I think we handle the basic hungryness you spawn with via

```
/datum/equipment_preset/uscm/load_status(mob/living/carbon/human/H)
	H.nutrition = rand(NUTRITION_VERYLOW, NUTRITION_LOW)
```

Reply:
Alright, I've tested how much impact it has and it basically makes you take extra 10 seconds to go from cryo to briefing food vendors. So overall it might be 1-2 minute detour after you've geared up, which is fine enough.

Still make it a part of the gear preset by removing this:
```
/datum/equipment_preset/uscm/intel/load_status()
	return //No cryo munchies
```